js/base/Exchange.js

```diff
@@ -1571,6 +1571,18 @@ module.exports = class Exchange {
         return ohlcv;
     }
 
+    parseOHLCVVs (ohlcvs, market = undefined, timeframe = '1m', since = undefined, limit = undefined) {
+        const results = [];
+        const useOHLCVV = this.safeValue (this.options, 'useNewOHLCV', false);
+        for (let i = 0; i < ohlcvs.length; i++) {
+            const ohlcvv = !useOHLCVV ? this.parseOHLCV (ohlcvs[i], market) : this.parseOHLCVV (ohlcvs[i], market);
```

Comment:
why do we need `useNewOHLCV`, aren't we going to completely separate `fetchOHLCV`and `fetchOHLCVV` ?